Ginger/Ginger/Repository/RepositoryItemFactory.cs

```diff
@@ -744,8 +744,16 @@ public void DownloadSolution(string SolutionFolder)
             {
                 sol.ExistInLocaly = true;
             }
-            else if (WorkSpace.Instance.UserProfile.SourceControlType == SourceControlBase.eSourceControlType.GIT && Directory.Exists(PathHelper.GetLongPath(sol.LocalFolder + @"\.git")))
+            else if (WorkSpace.Instance.UserProfile.SourceControlType == SourceControlBase.eSourceControlType.GIT)
             {
+                if (!sol.LocalFolder.EndsWith(".git"))
+                {
+                    Directory.Exists(PathHelper.GetLongPath(sol.LocalFolder + @"\.git"));
```

Comment:
Directory.Exist was in the if condition earlier.
Now regardless if directory exist or not sol.existLocaly will be always marked true if Souce control type is git